wuvt/admin/views.py

```diff
@@ -255,6 +255,18 @@ def page_edit(page_id):
                            page=page,
                            form=form)
 
+@bp.route('/page/draft/<int:page_id>')
+@auth_manager.check_access('admin', 'content')
+def page_draft(page_id):
+    page = Page.query.filter(Page.id == page_id).first()
+    if not page:
+        abort(404)
+
+    if not page.content:
+        page.content = page.summary
```

Comment:
Pages don't have summaries; only articles do. So we should probably throw an error here instead.

Other than that, looks good! I'll give it a test this evening.

Reply:
Sounds reasonable to me.